### DISCUSSION TITLE
Is it possible to set an item display name that is not italic ?

### DISCUSSION THREAD

**Discussion by bananasmoothii** [Minestom/Minestom#1306] [Help] [answered]
Whatever I try, items I give to players that have a custom display name have their display name in italics. On google I can find [that](https://www.reddit.com/r/MinecraftCommands/comments/e7t1f7/how_to_name_items_without_italics/) but I don't know how to implement that in minestom.
PS: I'm using KStom, does that matter ?

**Comment by LooFifteen** [comment, marked as answer]
Use `Component#decoration(TextDecoration.ITALICS, false)`.

**Reply by bananasmoothii** [reply]
Oh that was an easy fix !

**Comment by emortaldev** [comment]
KStom has `Component.noItalic()`

**Reply by bananasmoothii** [reply]
Oh, that is also useful, tanks !
